admin_manual/configuration/big_file_upload_configuration.rst

```diff
@@ -26,8 +26,10 @@ System Configuration
 Configuring Your Webserver
 --------------------------
 
-ownCloud comes with its own ``owncloud/.htaccess`` file. Set the following 
-two parameters inside this ``.htaccess`` file::
+| ownCloud comes with its own ``owncloud/.htaccess`` file.
+| If PHP-FPM is used, it can't read ``.htaccess`` PHP settings unless a PECL extension is installed. If PHP-FPM is used without the PECL extension installed, settings and permissions must be set in the ``owncloud/.user.ini`` file.
```

Comment:
As far as I understood, and please correct me if I am wrong, the line break is automatically done.

The reason why I used a hard linebreak at the scentence staring with "If PHP-FPM..." was, that readers have a better visibility to distinguish between standard and php-fpm. But I can remove that linebreak so all will be continious.
Or do you have any other place where you point to, and the what shall I do, because the text is in continious writing and linebreaks are made by the system.